lib/mirage.ml

```diff
@@ -1468,6 +1522,7 @@ module Project = struct
   let version = Mirage_version.current
   let prelude =
     "open Lwt\n\
+     module Logger(X:sig type t val run: t -> (unit -> 'a Lwt.t) -> 'a Lwt.t end) = X\n\
```

Comment:
I'm not sure where to put it.

Maybe there's a better alternative. I don't really want to define a new functor. I just want to know the name of the logging module, so that I can call `run` on it. This identity functor seems a bit of a hack, but I didn't see a better way